packages/expo-av/ios/EXAV/EXAV.m

```diff
@@ -541,7 +540,7 @@ - (NSDictionary *)_getAudioRecorderStatus
 - (BOOL)_checkAudioRecorderExistsOrReject:(UMPromiseRejectBlock)reject
 {
   if (_audioRecorder == nil) {
-    reject(@"E_AUDIO_NORECORDER", nil, UMErrorWithMessage(@"Recorder does not exist."));
+    reject(@"E_AUDIO_NORECORDER", @"Recorder does not exist.", nil);
```

Comment:
Same here.
I'm not sure what is causing this error (looking at this from the user perspective), but I wish I could narrow down the spectrum of possible inconsistencies causing to this error while debugging the JS code 🤔

Reply:
Yeah totally, these error could definitely be improved, and made consistent with the Android implementation. I've made a couple changes based on your suggestions so that it will be a bit clearer to the user what's going on in case these occur 👍